app/Observers/ContactObserver.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
What ContactObserver ?
These are maybe just specific modules, not related to this core. These things could be moved at UserObserver